spec/fixtures/ead/alphaomegaalpha.xml

```diff
@@ -83,11 +83,6 @@
         </defitem>
       </list>
     </odd>
-    <acqinfo id="aspace_4179cc95d0af3f4bc52ca7f563e45888">
-      <head>Provenance</head>
-      <p>Gift,John Z. Bowers in 1973; Gladys Brill Brampton in 1999 and 2000. Acc. 0169, Acc
-        1999-044, Acc 2000-012, Acc 2000-023.</p>
-    </acqinfo>
```

Comment:
We noticed this existed already in the fixture (but another `acqinfo` was added to satisfy the needs for this ticket).  

@anarchivist cool that we delete the original `acqinfo`?